### PR TITLE
(docs)(DOCUMENT-966) Warning for file backup docs

### PR DESCRIPTION
**Pull request by hestonhoffman**
This affects versions 5.5 and up.

https://tickets.puppetlabs.com/browse/DOCUMENT-966

### CODE REVIEW

File: lib/puppet/application/filebucket.rb [hunk 1]
```diff
@@ -69,6 +69,14 @@ def help
 use your local file bucket by specifying '--local', or by specifying
 '--bucket' with a local path.
 
+> **Note**: Enabling and using the backup option, and by extension the 
+  filebucket resource, requires appropriate planning and management to ensure 
+  that sufficient disk space is available for the file backups. Generally, you 
+  can implenent this using one of the following two options:
```
Comment (joshcooper) [inline lib/puppet/application/filebucket.rb]: nit `implement`, same comment in `file.rb`

**Comment by joshcooper** [comment]
Looks good. Is using `tidy` another recommended solution to clean up the directory? That may not work due to the state.yaml filling up with entries for files that no longer exist, until the state ttl hits in 32 days. /cc @tkishel @Sharpie

File: lib/puppet/application/filebucket.rb (hunk 1 above)
Comment (hestonhoffman) [reply lib/puppet/application/filebucket.rb]: Good catch! I'll push up an edit

**Comment by joshcooper** [comment]
@hestonhoffman sorry just saw your note about "versions 5.5 and up". If you want to update that version of the docs, then please retarget this PR for the 5.5.x branch in puppet, and it will auto-merge to other branches as CI passes. Thanks!

**Comment by tkishel** [comment]
> Looks good. Is using `tidy` another recommended solution to clean up the directory? That may not work due to the state.yaml filling up with entries for files that no longer exist, until the state ttl hits in 32 days.

Tidy (like File w `recurse`) will create a File Resource to all those files ... at minimum a Catch 22.

**Comment by hestonhoffman** [comment]
Closed in favor of #7992

**Pull request closed without merging**